site/docs/4.1/utilities/sizing.md

```diff
@@ -30,16 +30,22 @@ Width and height utilities are generated from the `$sizes` Sass map in `_variabl
 {% endcapture %}
 {% include example.html content=example %}
 
-You can also use `max-width: 100%;` and `max-height: 100%;` utilities as needed.
+You can also use `max-width` and `max-height` utilities as needed.
 
 {% capture example %}
-{% include icons/placeholder.svg width="100%" height="100" class="mw-100" text="Max-width 100%" title="Max-width 100%" %}
+<div class="mw-25 p-3" style="width: 100%; background-color: #eee;">Max-width 25%</div>
```

Comment:
You should probably use a loop here. Which reminds me, we should use a loop in the examples above these too.